diagnostics/microTCA-vs-BPMs-mapping/names_crate01.cfg

```diff
@@ -9,3 +9,4 @@ SI-01C3:DI-BPM-2            rfbpm-sr
 SI-01C4:DI-BPM              rfbpm-sr
 BO-01U:DI-BPM               rfbpm-boo
 BO-02U:DI-BPM               rfbpm-boo
+SI-01:BS-FOFBCtrl           fofb-ctrl
```

Comment:
The name of the device has the rack name: IA-01RaBPM:BS-FOFBCtrl

Reply:
right! I fixed it!